pkg/randstr/randstr_test.go

```diff
@@ -0,0 +1,15 @@
+package randstr_test
+
+import (
+	"testing"
+
+	"github.com/ansible/receptor/pkg/randstr"
+)
+
+func TestRandStr(t *testing.T) {
+	randomStr := randstr.RandomString(10)
```

Comment:
Lets wrap this in a `t.Run()` and lets add edge cases to this please.

What happens when we pass in 0, -1, and 100. Also are the return values truely random, lets check to make sure the return string is never the seed `"0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"`